contact.css

```diff
@@ -0,0 +1,225 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300:400');
+
+*, *:before, *:after {
+	box-sizing: border-box;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+  }
+  
+  body {
+	background: linear-gradient(
+		60deg,
+		rgba(84, 58, 183, 1) 0%,
+		rgba(0, 172, 193, 1) 100%);
+	font-size: 12px;
+  }
+  
+  body, button, input {
+	font-family: "Lato", sans-serif;
+	font-weight: 700;
+	letter-spacing: 1.4px;
+  }
+  
+  .background {
+	display: flex;
+	min-height: 100vh;
+  }
+  
+  .container {
+	flex: 0 1 700px;
+	margin: auto;
+	padding: 10px;
+  }
+  
+  .screen {
+	position: relative;
+	background: #3e3e3e;
+	border-radius: 15px;
+  }
+  
+  .screen:after {
+	content: '';
+	display: block;
+	position: absolute;
+	top: 0;
+	left: 20px;
+	right: 20px;
+	bottom: 0;
+	border-radius: 15px;
+	box-shadow: 0 20px 40px rgba(0, 0, 0, .4);
+	z-index: -1;
```

Comment:
- Will it be visible when you put -1 as index please cross check I am not sure

Reply:
It is working I guess